### PR TITLE
Backend api routes + Stripe integration 

### PR DESCRIPTION
**Pull request by davidzny195**

### CODE REVIEW

**Review by BrettEastman — APPROVED** [review]
Looks great, David! The only thing I would suggest is to push an example.env file for everyone

File: src/server/models/admin.js [hunk 1]
```diff
@@ -1 +1,62 @@
...
+  },
+  post: async (payload) => {
+    try {
+      const db = await dbPromise;
+      const superUsersRef = collection(db, 'superUsers');
+
+      return setDoc(doc(superUsersRef), {
+        ...payload,
+        createdAt: serverTimestamp(),
+        updatedAt: serverTimestamp(),
```
Comment (kevinleaves) [inline src/server/models/admin.js:38]: Very cool you added the timestamp using Firestore's serverTimestamp(). I'll look into that.

File: src/server/routes/schedules.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router();
+const {
+  scheduleController: { get, getOne, post, patch, remove },
+} = require('../controllers');
```
Comment (kevinleaves) [inline src/server/routes/schedules.js:4]: nice use of destructuring in the routes. I'll make sure to do this in the future for sure.

File: src/client/pages/Payment.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        <img
+          src="https://i.imgur.com/EHyR2nP.png"
+          alt="The cover of Stubborn Attachments"
+        />
+        <div className="description">
+          <h3>Stubborn Attachments</h3>
+          <h5>$20.00</h5>
+        </div>
+        <form
+          action="http://localhost:8080/api/checkout"
```
Comment (kevinleaves) [inline src/client/pages/Payment.jsx:40]: Haven't seen the action attribute used in a form element before. Will look into it.

**Pull request merged**